vrc-get-gui/src/templates.rs

```diff
@@ -472,22 +490,39 @@ pub async fn create_project(
                     .alcom_template
                     .as_ref()
                     .expect("no .alcomtemplate info");
+                log::debug!("Resolving custom template: {} based on {}", id, template.base);
+
                 let mut resolved = resolve_template(templates, &template.base, unity_version)?;
 
+                // Merge VPM dependencies
                 for (pkg_id, range) in &template.vpm_dependencies {
                     match resolved.packages.entry(pkg_id.clone()) {
                         Entry::Occupied(mut e) => {
-                            let range = range.intersect(e.get());
-                            e.insert(range);
+                            log::debug!(
+                                "Merging existing dependency: {} (Base: {}, Custom: {})",
+                                pkg_id,
+                                e.get(),
+                                range
+                            );
+                            let intersected_range = range.intersect(e.get());
```

Comment:
It might be better (and enough) to log intersected_range and not log base / custom since those templates are likely to be already resolved and logged before.
```suggestion
                            let intersected_range = range.intersect(e.get());
                            log::debug!(
                                "Merged existing dependency: {} {}",
                                pkg_id,
                                intersected_range,
                            );
```

Reply:
Also this. no response

Reply:
As mentioned, these should have been addressed via VS, but they somehow didn't post. I'll get back to you

Reply:
I've revisited and will recommit with the suggested changes. An oversight on my end since it was a part of a debugging i did.

Reply:
Updated.